plugins/cockroachdb_logs.yaml

```diff
@@ -0,0 +1,317 @@
+parameters:
+  - name: enable_health_log
+    description: Enable to collect health logs
+    type: bool
+    default: true
+  - name: health_log_path
+    description: The absolute path to the CockroachDB health logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-health.log"
+  - name: enable_dev_log
+    description: Enable to collect general Dev logs.
+    type: bool
+    default: true
+  - name: dev_log_path
+    description: The absolute path to the CockroachDB Dev Logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach.log"
+  - name: enable_error_log
+    description: Enable to collect stderr logs.
+    type: bool
+    default: true
+  - name: error_log_path
+    description: The absolute path to the CockroachDB stderr logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-stderr.log"
+  - name: enable_sql_schema_log
+    description: Enable to collect sql schema logs.
+    type: bool
+    default: true
+  - name: sql_schema_log_path
+    description: The absolute path to the CockroachDB sql schema logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-sql-schema.log"
+  - name: enable_telemetry_log
+    description: Enable to collect telemetry logs.
+    type: bool
+    default: true
+  - name: telemetry_log_path
+    description: The absolute path to the CockroachDB telemetry logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-telemetry.log"
+  - name: enable_kv_distribution_log
+    description: Enable to collect kv distribution logs.
+    type: bool
+    default: true
+  - name: kv_distribution_log_path
+    description: The absolute path to the CockroachDB kv distribution logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-kv-distribution.log"
+  - name: enable_pebble_log
+    description: Enable to collect cockroachdb pebble logs.
+    type: bool
+    default: true
+  - name: pebble_log_path
+    description: The absolute path to the CockroachDB pebble logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-pebble.log"
+  - name: start_at
+    description: At startup, where to start reading logs from the file ('beginning' or 'end')
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end 
+  - name: timezone
+    description: Timezone to use when parsing the timestamp
+    type: timezone
+    default: UTC
+  - name: retain_raw_logs
+    description: When enabled will preserve the original log message in a `raw_log` key. This will either be in the `body` or `attributes` depending on how `parse_to` is configured.
+    type: bool
+    default: false
+  - name: parse_to
+    description: Where to parse structured log parts
+    type: string
+    supported:
+      - body
+      - attributes
+    default: body
+  - name: offset_storage_dir
```

Comment:
This needs to be set as an extension before the receivers like [here](https://github.com/observIQ/observiq-otel-collector/blob/release/v1.26.0/plugins/iis_logs.yaml#L72-L74). Then is needs to be referenced in each `filelog` receiver like [here](https://github.com/observIQ/observiq-otel-collector/blob/release/v1.26.0/plugins/iis_logs.yaml#L79)